Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```
@@ -845,10 +852,18 @@
     fireModes:
     - proto: BulletLaserSpread
       fireCost: 150
+      color: "#5fcde4"
     - proto: BulletLaserSpreadNarrow
       fireCost: 200
+      color: "#5fcde4"
     - proto: BulletDisablerSmgSpread
       fireCost: 120
+      color: "#ff0006"
```

```suggestion
    - proto: BulletLaserSpread
      fireCost: 150
      color: "#ff0006"
    - proto: BulletLaserSpreadNarrow
      fireCost: 200
      color: "##ff9d00"
    - proto: BulletDisablerSmgSpread
      fireCost: 120
      color: "#5fcde4"
```
The colors are set wrong, and I recommend giving the narrow fire mode its own color. In this case I gave it an orange to set it apart from the wide.